src/utils/ga.ts

```diff
@@ -86,9 +86,9 @@ export const getGaTrackUrl = <State>(
 ) => {
     const gaConfig = isGaEnabled(config)
         ? typeof config.googleAnalytics === "boolean"
-            ? defaultConfig.googleAnalytics
+            ? getDefaultConfig().googleAnalytics
```

Comment:
getDefaultConfig() should only be used to merge defaults into the full configuration